src/content/docs/en/guides/astro-db.mdx

```diff
@@ -654,6 +655,68 @@ When you're ready to deploy, see our [Deploy with a Studio Connection guide](#de
 
 </ReadMore>
 
+## LibSQL
+
+<p><Since pkg="@astrojs/db" v="0.14.0" /></p>
+
+Astro DB can connect to any LibSQL server from any platform that exposes the Hrana protocol of the server, or host it yourself.
+
+To connecto Astro DB to a LibSQL instead of Studio, set the following environment variables:
```

Comment:
```suggestion
To connect Astro DB to a LibSQL instance, set the following environment variables:
```
What's the noun here, is it "instance"?

Reply:
I think "database" would be the most appropriate. At first I thought of server but it can be a local file or even in memory.